proposals/wg-reference-types.md

```diff
@@ -0,0 +1,86 @@
+# Working Group Proposal: Reference Types
+
+Proposal created from [OCI WG template](https://github.com/opencontainers/tob/blob/master/WG-TEMPLATE.md).
+
+Proposal copied from [Proposal: Working Group for Reference Types #96](https://github.com/opencontainers/tob/issues/96)
+
+## Reference Types OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the Reference Types Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+As cloud native development continues to grow, there is increased interest in evolving registries to natively store, discover, and pull a graph of content associated with specific container images in a registry. Use cases for said associated artifacts include but are not limited to Software Bill of Materials (SBoM), security scan results, and signing. Having a native way to store and discover artifacts associated with other artifacts enables end-users to answer the question of: “What SBOMs or signatures are associated with this container image?”
+
+## Scope
+
+* Define and deliver the capability to store, discover, and pull a graph of artifacts associated with a specific artifacts to OCI distribution compliant registries. These set of capabilities has been commonly known as "reference types" or "references".
+  * Define supported use cases
+  * Document impact to existing user-facing tools and registries
+  * Define the method for creating, distributing, and discovering referenced objects
+  * Document user expectations for promoting an artifact between registries with its references
+  * Document onboarding process for registries and user-facing tools to adopt reference types
+  * Defined expectations for artifact reference lifecycle management
+  * Deliver a reference implementation of the reference types proposal
+
+## Out of Scope
+
+* Identified out of scope items will be listed here as WG progresses
+
+## Intended work product
+
+* Referrers API specification that provides the ability to discover references to existing container images. These include listing signatures, SBoMs, security scan results that refer to the digest of a manifest. The referrers API specification will sit within or along side the Distribution specification.
+* Identify, and document the pros and cons for versioning the existing manifests, compared with creating a new manifest to support reference types.
```

Comment:
```suggestion
* Proposal to extend existing manifests or create a new manifest to support reference types.
```

Reply:
To address any of the existing manifests, we do need to address how these would be versioned. So, I propose we leave this as stated as the updated wording does change the process.

Reply:
Just documenting the pros and cons doesn't feel like a great work product. Having a concrete proposal for existing manifests feels like a prerequisite for doing this analysis, so if that's implied by what you have here, that's fine with me.

Reply:
Ahh, I see, so it's a merger of the two. 
What I didn't want to lose was the versioning analysis, and just jump into debating solutions. I've merged both concepts.